client/src/components/SponsorsComps/HomeCommunitySponsors.js

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable prettier/prettier */
+import React from 'react';
+import { hyperlink } from "../../utils";
+import { image } from "../../images";
+import styled from "styled-components";
+
+// Styled-component Section
+
+const HomeCommunityContainer = styled.div`
```

Comment:
OK, Important note: I am not doing this review chronologically 🙃meaning: I am going over every file from top to bottom, but I reviewed `HomePageSponsorsSection.jsx` first

So, suggested there "a different naming + breakdown" , there is a picture there I attached, and an example of jsx I wrote somewhere below - all in the sam file

Now, what I want to say here is that:


In `HomePageSponsorsSection.jsx`, there is 

```
<CommunitySponsorsSection>
          <HomeCommunitySponsors sponsors={sponsors.filter(sponsor => !sponsor.isPremium)}/></CommunitySponsorsSection>
```

Now, `CommunitySponsorsSection` has the following css

```
display: flex;
justify-content: space-between;
flex-wrap: wrap;
```

Now, in this file, you have a `HomeCommunityContainer`, which has same css

```
display: flex;
justify-content: space-between;
flex-wrap: wrap;
```

First of all, its redundant : )))

Second, `Section` and `Container` has a similar meaning in a way... Not necessarily, but lets have a consistent way of calling something that creates the **layout**  (flex stuf/ max-width...) for the component we include. `Container` works for me.

Third, Either we move the containers to the parent file, or have the container in each file, so it is self contained, better I think, usually.

Reply:
I'm on it.
Regarding naming, `Section`s are `<section>` and containers are `<div>`s.